cinema/views.py

```diff
@@ -46,7 +52,7 @@ class ActorViewSet(
 ):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
-    authentication_classes = (TokenAuthentication,)
+
```

Comment:
The `permission_classes` attribute is correctly set to `IsAdminOrIfAuthenticatedReadOnly`, which aligns with the intended access control for this viewset.